chains/tendermint/prover.go

```diff
@@ -187,20 +192,19 @@ func (pr *Prover) UpdateLightWithHeader() (header core.HeaderI, provableHeight i
 		}
 	}
 
-	protoVal, err := tmtypes.NewValidatorSet(sh.ValidatorSet.Validators).ToProto()
+	valSet := tmtypes.NewValidatorSet(sh.ValidatorSet.Validators)
+	protoVal, err := valSet.ToProto()
 	if err != nil {
 		return nil, 0, 0, err
 	}
+	protoVal.TotalVotingPower = valSet.TotalVotingPower()
 
 	h := &tmclient.Header{
 		SignedHeader: sh.SignedHeader.ToProto(),
 		ValidatorSet: protoVal,
 	}
-	queryableHeight = int64(h.GetHeight().GetRevisionHeight())
-	// NOTE: We query connection at height - 1 because of the way tendermint returns
-	// proofs the commit for height n is contained in the header of height n + 1
-	provableHeight = queryableHeight - 1
-	return h, provableHeight, queryableHeight, nil
+	height := int64(h.GetHeight().GetRevisionHeight())
+	return h, height, height, nil
```

Comment:
Does this modification follow changes of implementation of Tendermint?

Reply:
yeah, this is due to cosmos-sdk updates: https://github.com/cosmos/cosmos-sdk/pull/9879